packages/wells/src/client/api/wellboresAPI.ts

```diff
@@ -0,0 +1,30 @@
+import { CDFHttpClient} from '@cognite/sdk-core';
+import { Survey } from '../model/Survey';
+
+export class WellboresAPI {
+  private client?: CDFHttpClient;
+  private project?: String;
+
+  public set setHttpClient(httpClient: CDFHttpClient) {
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  /* eslint-disable */
+  public getTrajectory = async (wellboreId: number): Promise<Survey | undefined> => {
+      const path: string = `/${this.project}/wellbores/${wellboreId}/trajectory`
+      let trajectory: Survey | undefined;
+      await this.client?.get<Survey>(path)
+      .then(response => {
+        if (response.status == 200){
+          trajectory = response.data;
+        } else {
+          trajectory = undefined;
```

Comment:
here we will be hiding the error message received from the API, perhaps we could have some logging or maybe throw an error to help the user debug?